docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -83,6 +83,31 @@ A 4th node would push your total memory allocation to 64 GiB, violating your qu
 
 If necessary, be sure to request a quota increase via our [{{support}}](https://{{support_domain}}/servicedesk).
 
+## Viewing details and monitoring
+
+After the new shoot cluster finishes bootstrapping, you may click on its row to bring all relevant details into view.
+For instance, click the *Details* tab to get networking information about the worker nodes.
+
+![Networking information about the worker nodes](assets/create-shoot-09.png)
+
+Next, go to the *Monitoring* tab.
+There, you will notice two orange buttons, each for launching [*Prometheus*](https://prometheus.io) or [*Plutono*](https://github.com/credativ/plutono).
```

Comment:
The rest of the guide uses weak emphasis (italics) for UI labels. Since we have no tab or button labeled, literally, "Prometheus" or "Plutono", I'd suggest dropping the italics here:

```suggestion
There, you will notice two orange buttons, each for launching [Prometheus](https://prometheus.io) or [Plutono](https://github.com/credativ/plutono).
```

Reply:
On second thought, there's another modification I'd suggest:

```suggestion
There, you will notice two orange buttons: one for launching [Prometheus](https://prometheus.io), and one for [Plutono](https://github.com/credativ/plutono).
```

Reply:
Ah, yes, better.